draft-wood-tls-external-psk-importer.md

```diff
@@ -97,20 +100,29 @@ supported HashAlgorithm 'hash', the importer constructs an ImportedIdentity stru
    } ImportedIdentity;
 ~~~
 
+ImportedIdentity.label SHOULD be bound to the protocol for which the key is imported. For example,
+TLS 1.3 over TCP should use "tls13" as the label, whereas QUIC {{!I-D.ietf-quic-transport}} should use
+"quicv1" as the label. Note that this means future versions of TLS will increase the number of
+PSKs derived from an external PSK.
+
 A unique and imported PSK (IPSK) with base key 'ipskx' bound to this identity is then computed as follows:
 
 ~~~
    epskx = HKDF-Extract(0, epsk)
-   ipskx = HKDF-Expand-Label(epskx, "derived psk", Hash(ImportedIdentity), Hash.length)
+   ipskx = HKDF-Expand-Label(epskx, "derived psk",
+                             Hash(ImportedIdentity), Hash.length)
```

Comment:
Should `Hash.length` be instead the PSK size that the particular IPSK use wants? I suppose TLS 1.3 doesn't really care, but if the goal is to support other protocols, they may have opinions on key size. (Maybe it becomes an AES key later, I dunno.)

Reply:
Good point! I'll generalize it to be the length of the CipherSuite key, with a note that says this is the same as Hash.length in 1.3.